CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 2.3.3
+
+* **[customLocator plugin](#customlocator) introduced**. Adds a locator strategy for special test attributes on elements.
+
+```js
+// when data-test-id is a special test attribute
+// enable and configure plugin to replace this
+I.click({ css: '[data-test-id=register_button]');
+// with this
+I.click('$register_button');
+```
+* [Puppeteer][WebDriver] `pressKey` improvements by @martomo:
+Changed pressKey method to resolve issues and extend functionality.
+  * Did not properly recognize 'Meta' (or 'Command') as modifier key.
+  * Right modifier keys did not work in WebDriver using JsonWireProtocol.
+  * 'Shift' + <key> combination would not reflect actual keyboard behavior.
+  * Respect sequence with multiple modifier keys passed to pressKey.
+  * Added support to automatic change operation modifier key based on operating system.
+* [Puppeteer][WebDriver] Added `detectOS` config parameter which automatically changes `Control` to `Meta` when pressing key on macOS. And changes `Meta` to `Control` when running on Windows or Linux.
```

Comment:
@DavertMik I did some research/thinking and perhaps instead of using the `detectOS` (in helper/test config), I would like to change this to a `'CommandOrControl'` alias (similar implementation as for [Electron](https://electronjs.org/docs/api/accelerator#platform-notice)) before releasing 2.3.3.